fake-files/output-expected/spraggins/spraggins.ims.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dimensions": [
+    "mz",
+    "x",
+    "y"
+  ],
+  "zarrSource": "https://s3.amazonaws.com/vitessce-data/0.0.20/master_release/spraggins/spraggins.ims.zarr"
```

Comment:
I know I called it `tileSource`, but do you think `zarrUrl` would be less ambiguous?